_posts/2024-05-14-16.1.0-release.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: "Apache Arrow 16.1.0 Release"
+date: "2024-05-14 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 16.1.0 release.
+This is a minor release that includes [**34 resolved issues**][1]
+from [**16 distinct contributors**][2]. See the Install Page to learn how to get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## C Data Interface notes
+
+
+## Arrow Flight RPC notes
+
+
+## C++ notes
+
+
+## C# notes
```

Comment:
@CurtHagenlocher @adamreeve can you help adding the C# notes for the 16.1.0 release?

Reply:
How about:
```suggestion
## C# notes
* Recompute a sliced array's null count on demand when it is unknown ([GH-41136](https://github.com/apache/arrow/issues/41136))
* Support writing sliced arrays in the Arrow IPC format ([GH-40517](https://github.com/apache/arrow/issues/40517), [GH-41225](https://github.com/apache/arrow/issues/41225), [GH-41231](https://github.com/apache/arrow/issues/41231))
* Bug fixes for union array behaviour ([GH-41137](https://github.com/apache/arrow/issues/41137), [41140](https://github.com/apache/arrow/issues/41140))
```